### DISCUSSION TITLE
__nothing__

### DISCUSSION THREAD

**Discussion by kikiru328** [kikiru328/Bone_Detection#31] [CHAT]
__nothing__

**Comment by kikiru328** [comment]
https://user-images.githubusercontent.com/60537388/138037684-7fb098af-aaf7-4e74-97f0-f1bec72a10bf.mp4

**Reply by SHEYCHAE** [reply]
매롱

**Comment by SHEYCHAE** [comment]
<img src= "https://user-images.githubusercontent.com/77530039/138319580-7321c5cb-9d0d-4a01-a5d5-94c95a07ccb4.png" width="600" height="400"/> 
labeling 해본거 (이렇게하면 안됨 전처리과정 다 빼고  그냥 연습해본거)
